enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/components/Visualization.tsx

```diff
@@ -33,19 +37,23 @@ export const QuestionVisualization = () => {
   const card = question.card();
 
   return (
-    <QueryVisualization
-      className={cx(CS.flexFull, CS.fullWidth, CS.fullHeight)}
-      question={question}
-      rawSeries={[{ card, data: result && result.data }]}
-      isRunning={isQueryRunning}
-      isObjectDetail={false}
-      isResultDirty={false}
-      isNativeEditorOpen={false}
-      result={result}
-      noHeader
-      mode={mode}
-      navigateToNewCardInsideQB={onNavigateToNewCard}
-      onNavigateBack={onNavigateBack}
-    />
+    <Box ref={ref} w="100%" h="100%">
```

Comment:
I'm trying something here - instead of forcing a user to explicitly set a height and width, I'm trying to see if we can get the visualization to match the height and width of its parent `div`. As a FE that's exactly what I would want, since I might have a responsive layout and I _really_ hate dealing with height/width values. If you look at the demo, the visualization is using this logic and I think it's working pretty well. Let me know what you think 😄